src/ios/THEThaliCore.m

```diff
@@ -122,8 +122,8 @@ - (CDVPlugin*)initWithWebView:(UIWebView *)theWebView
 // Define extensions.
```

Comment:
Is there any reason for this file to exist anymore? It looks like test code. But I'll be the first to admit that I don't fully understand how JXcore hooks everything together.